timescaledb/how-to-guides/migrate-data/different-db.md

```diff
@@ -0,0 +1,75 @@
+# Migrating from a Different PostgreSQL Database
```

Comment:
Probably we should normalize the headers as we did on #39. WDYT?

```suggestion
# Migrating from a different PostgreSQL database

```

Reply:
yep, agreed 👍